src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
**Use ES6 modules to write modular JavaScript with proper ES6 syntax.**

- Usage of arrow functions instead of the `` function `` keyword.
- usage of `` import `` and  `` export ``  instead of `` require `` or `` module.exports``.
- proper usage of  ``const``  and `` let``  keywords.

Reply:
Thank you for your review.
But can you clarify more your review, since I see nothing to change in this file, and for milestone 1 , I need only to implement the main structure using HTML.
Thank you in advance.

Reply:
> **Use ES6 modules to write modular JavaScript with proper ES6 syntax.**
> 
> * Usage of arrow functions instead of the `function` keyword.
> * usage of `import` and  `export`  instead of `require` or ` module.exports`.
> * proper usage of  `const`  and ` let`  keywords.

Thank you for your review.
But can you clarify more your review, since I see nothing to change in this file, and for milestone 1 I need only to implement the main structure using HTML.
Thank you in advance.